source/common/common/perf_annotation.cc

```diff
@@ -108,7 +108,7 @@ std::string PerfAnnotationContext::toString() {
             : std::to_string(
                   std::chrono::duration_cast<std::chrono::nanoseconds>(stats.total_).count() /
                   count));
-    columns[3].push_back(fmt::format("{}", stats.stddev_.computeStandardDeviation()));
+    columns[3].push_back(absl::StrCat("", stats.stddev_.computeStandardDeviation()));
```

Comment:
"" as the first argument to absl::StrCat seems unnecessary.

Reply:
you're right